operation/manager_test.go

```diff
@@ -176,5 +187,5 @@ func (m *mock) stopFail(ctx context.Context, extra map[string]interface{}) error
 }
 
 func (m *mock) IsPowered(ctx context.Context, extra map[string]interface{}) (bool, float64, error) {
-	return false, 0, nil
+	return true, 1, nil
```

Comment:
One more time: you're sure this line needed changing? I don't see a call to `IsPowered` anywhere in this file. which doesn't mean it's not used, just that it's hard for me to see that.